package/src/components/KeyboardCompatibleView/KeyboardCompatibleView.tsx

```diff
@@ -178,12 +180,17 @@ export class KeyboardCompatibleView extends React.Component<
   };
 
   componentDidMount() {
-    AppState.addEventListener('change', this._handleAppStateChange);
+    this._appStateSubscription = AppState.addEventListener('change', this._handleAppStateChange);
     this.setKeyboardListeners();
   }
 
   componentWillUnmount() {
-    AppState.removeEventListener('change', this._handleAppStateChange);
+    // Following if-else condition to avoid deprecated warning coming RN 0.65
```

Comment:
Due to this - https://github.com/react-native-community/releases/blob/master/CHANGELOG.md#:~:text=EventEmitter%23removeSubscription%20is%20now%20deprecated.%20(cb6cbd12f8%20by%20%40yungsters)